dashboard/src/containers/RoutesContainer/Routes.tsx

```diff
@@ -47,8 +61,10 @@ const privateRoutes = {
     OperatorInstanceViewContainer,
   "/c/:cluster/ns/:namespace/operators-instances/:csv/:crd/:instanceName/update":
     OperatorInstanceUpdateContainer,
-  "/c/:cluster/ns/:namespace/config/repos": AppRepoList,
-  "/docs": ApiDocs,
+} as const;
+
+const unsupportedRoutes = {
+  "/c/:cluster/ns/:namespace/operators*": "Operators support has been disabled for Kubeapps.",
```

Comment:
Perhaps: "Operators support has been disabled by default for Kubeapps. Enable it in your values.yaml" (or something better, that sounds awkward, but something communicating that it can be enabled).